test/UserRepository/PdoDatabaseTest.php

```diff
@@ -69,36 +75,63 @@ public function testAuthenticateInvalidUsername()
     public function testAuthenticateWithRole()
     {
         $pdo = new PDO('sqlite:'. __DIR__ . '/../TestAssets/pdo_role.sqlite');
-        $pdoDatabase = new PdoDatabase($pdo, [
-            'table' => 'user',
-            'field' => [
-                'username' => 'username',
-                'password' => 'password',
-            ],
-            'sql_get_roles' => 'SELECT role FROM user WHERE username = :username'
-        ]);
+        $config = $this->getConfig();
+        $config['sql_get_roles'] = 'SELECT role FROM user WHERE username = :identity';
+        $pdoDatabase = new PdoDatabase($pdo, $config);
 
         $user = $pdoDatabase->authenticate('test', 'password');
         $this->assertInstanceOf(UserInterface::class, $user);
-        $this->assertEquals('test', $user->getUsername());
+        $this->assertEquals('test', $user->getIdentity());
         $this->assertEquals(['admin'], $user->getUserRoles());
     }
 
     public function testAuthenticateWithRoles()
     {
         $pdo = new PDO('sqlite:'. __DIR__ . '/../TestAssets/pdo_roles.sqlite');
-        $pdoDatabase = new PdoDatabase($pdo, [
-            'table' => 'user',
-            'field' => [
-                'username' => 'username',
-                'password' => 'password',
-            ],
-            'sql_get_roles' => 'SELECT role FROM user_role WHERE username = :username'
-        ]);
+        $config = $this->getConfig();
+        $config['sql_get_roles'] = 'SELECT role FROM user_role WHERE username = :identity';
+        $pdoDatabase = new PdoDatabase($pdo, $config);
 
         $user = $pdoDatabase->authenticate('test', 'password');
         $this->assertInstanceOf(UserInterface::class, $user);
-        $this->assertEquals('test', $user->getUsername());
+        $this->assertEquals('test', $user->getIdentity());
         $this->assertEquals(['user', 'admin'], $user->getUserRoles());
     }
+
+    /**
+     * @expectedException Zend\Expressive\Authentication\Exception\RuntimeException
```

Comment:
as above, please use method just before line where the exception should be thrown